ipam/space/space.go

```diff
@@ -122,8 +122,11 @@ func (s *Space) biggestFreeRange(r address.Range) (biggest address.Range) {
 	biggestSize := address.Count(0)
 	s.walkFree(r, func(chunk address.Range) bool {
 		if size := chunk.Size(); size >= biggestSize {
-			biggest = chunk
-			biggestSize = size
+			chunk = chunk.BiggestPow2AlignedRange()
```

Comment:
The biggest chunk does not guarantee that it contains the biggest CIDR-aligned range.

If we don't care too much about a few CPU cycles being wasted, then I'd suggest to merge the if-statements.